backend/src/dao/CoffeeChatDao.ts

```diff
@@ -172,4 +173,16 @@ export default class CoffeeChatDao extends BaseDao<CoffeeChat, DBCoffeeChat> {
   static async deleteMemberProperties(email: string): Promise<void> {
     memberPropertiesCollection.doc(email).delete();
   }
+
+  /**
+   * Gets a mapping of all categories to corresponding members, given the semester
```

Comment:
Maybe add a note here that this data could be up to X number of days stale? So changes made to member info or category info aren't immediately reflected here.

Also, how is this data populated initially? It might be good to have a script that refreshes this data once a day.